README_OSE.md

```diff
@@ -129,7 +129,7 @@ On the master host:
 ```sh
 oadm registry --create=true \
   --credentials=/etc/openshift/master/openshift-registry.kubeconfig \
-  --images='rcm-img-docker01.build.eng.bos.redhat.com:5001/openshift3/ose-${component}:${version}' \
+  --images='registry.access.redhat.com:5001/openshift3/ose-${component}:${version}' \
```

Comment:
Remove the 5001 port? Also as of 3.0.1 this shouldn't be necessary, perhaps remove the entire --images line?